include/alpaka/mem/fence/MemFenceCpu.hpp

```diff
@@ -54,13 +54,10 @@ namespace alpaka
                  *   a == 10 && b == 20
                  */
 
-                static auto dummy = std::atomic<int>{42};
-
-                /* ISO C++ fences are only clearly defined if there are atomic operations surrounding them. So we use
-                 * these dummy operations to ensure this.*/
-                auto x = dummy.load(std::memory_order_relaxed);
+                /* ISO C++ fences are only clearly defined if there are atomic operations surrounding them.
+                 * When using the fence, ensure that this condition is met in the code.
+                 */
```

Comment:
Please also drop this comment, since it's no longer accurate. Fences still need atomics, but the user must now provide them. You may add a documentation comment somewhere else, but not her :)
```suggestion
```